downstairs/src/repair.rs

```diff
@@ -226,7 +234,12 @@ fn extent_file_list(
     eid: u32,
 ) -> Result<Vec<String>, HttpError> {
     let mut files = Vec::new();
-    let possible_files = vec![(extent_file_name(eid, ExtentType::Data), true)];
+    let possible_files = vec![
+        (extent_file_name(eid, ExtentType::Data), true),
+        (extent_file_name(eid, ExtentType::Db), false),
+        (extent_file_name(eid, ExtentType::DbShm), false),
+        (extent_file_name(eid, ExtentType::DbWal), false),
+    ];
```

Comment:
Two nits:
- This could be an array instead of a `Vec`
- No need to call `extent_file_name(eid, ..)` here, just do it in the loop instead

```suggestion
    let possible_files = [
        (ExtentType::Data, true),
        (ExtentType::Db, false),
        (ExtentType::DbShm, false),
        (ExtentType::DbWal, false),
    ];
```

Reply:
Done in [7bef555](https://github.com/oxidecomputer/crucible/commit/7bef55560bd1a345855ca77b6c296616ee66e0fe)